### ISSUE TITLE
Rework profile links a bit 

### ISSUE THREAD

**Issue by yangwao** [kodadot/nft-gallery#2220]
At a collection view 
- replace subscan icon with identicon
- remove dot scanner link

At a profile
On left
- remove dotscanner and sub.id

On right, add over sharing, in same style into boxes
- dotscanner
- sub.id
- subscan
- polkascan

https://kusama.polkastats.io/account/Hqa6PRbEH25UiaPmcfeCfdG8FSqBUtNR5KtVfG3oSCBFSg9

![image](https://user-images.githubusercontent.com/5887929/152530285-956997ad-566f-43b4-a62d-0a5e3c4f92a9.png)

**Comment by prachi00** [comment]
@yangwao whats identicon?

**Comment by kkukelka** [comment]
@yangwao do you mean something like this for the right part of user profile?
![Screenshot 2022-02-07 at 08 22 53](https://user-images.githubusercontent.com/17953978/152742985-6bb5bae1-f7fc-4ed0-a7ae-0fcea2077ca7.png)

**Comment by yangwao** [comment]
> @yangwao do you mean something like this for the right part of user profile? ![Screenshot 2022-02-07 at 08 22 53](https://user-images.githubusercontent.com/17953978/152742985-6bb5bae1-f7fc-4ed0-a7ae-0fcea2077ca7.png)

Yup, but could be the same size as that icon and best if it would be merged together? Like grid of two rows

**Comment by yangwao** [comment]
OR we can think of using some dropdown or hiding it somehow smart and show on demand.

**Comment by kkukelka** [comment]
> Yup, but could be the same size as that icon and best if it would be merged together? Like grid of two rows

Yes will do. I actually like two rows in same grid with same-sized items the most. Since accordion approach would demand even more space.

**Comment by yangwao** [comment]
> > Yup, but could be the same size as that icon and best if it would be merged together? Like grid of two rows
> 
> Yes will do. I actually like two rows in same grid with same-sized items the most. Since accordion approach would demand even more space.

yes, that was ours first iteration, let's see how it would look.

we have still in mind that we don't need to show some icons until requested by the user so itentively hiding is the way to go. to keep UI as light as possible )
